deebot_client/util.py

```diff
@@ -88,6 +88,55 @@ def __hash__(self) -> int:
         return hash(self._value_)
 
 
+@unique
+class DisplayNameStrEnum(Enum):
```

Comment:
Why are you adding this enum back?

Reply:
because inside the **event auto_empty.py** in class **AutoEmptyMod** i used instead of **integer enums** **string enums**. For that I copy the existing **int enum** definition to have it with same functions for **string enum**.